ta/pkcs11/src/token_capabilities.c

```diff
@@ -225,6 +238,30 @@ void mechanism_supported_key_sizes(uint32_t proc_id, uint32_t *min_key_size,
 				   uint32_t *max_key_size)
 {
 	switch (proc_id) {
+	case PKCS11_CKM_MD5_HMAC:
+		*min_key_size = 16;
```

Comment:
Why not use TEE_MD5_HASH_SIZE and friends in here like what is on input_sign_size_is_valid()?

Reply:
> Why not use TEE_MD5_HASH_SIZE and friends in here like what is on input_sign_size_is_valid()?

I can do that but the min and max key sizes over here are not accurate which I am planning to replace by another PR. I left them here to be consistent with the sks code from @etienne-lms . The Global Platform Internal API used for HMAC support different min and max key sizes not limited to hash digest size.  Once this PR is merged, I plan to submit another PR for the same.